waiter/src/waiter/core.clj

```diff
@@ -737,8 +737,8 @@
                          (let [local-router (InetAddress/getLocalHost)
                                waiter-router-hostname (.getCanonicalHostName local-router)
                                waiter-router-ip (.getHostAddress local-router)
-                               ;; use (set [...]) instead of #{...} below as there may be duplicate values
-                               hostnames (set/union waiter-hostnames (set [waiter-router-hostname waiter-router-ip]))]
+                               ;; use (hash-set ...) instead of #{...} below, as there may be duplicate values
+                               hostnames (apply hash-set waiter-router-hostname waiter-router-ip waiter-hostnames)]
```

Comment:
How about `(into waiter-hostnames  [waiter-router-hostname waiter-router-ip])`? Then we no longer need the comment.

Reply:
I like that better. This line was affected by the rebase. Previously, `waiter-hostnames` wasn't a set, so this made more sense.

Reply:
Okay.

Reply:
I ended up changing it to `conj` instead.